pkg/v1/remote/transport/error.go

```diff
@@ -26,6 +26,10 @@ import (
 // https://github.com/docker/distribution/blob/master/docs/spec/api.md#errors
 type Error struct {
 	Errors []Diagnostic `json:"errors,omitempty"`
+	// The http status code returned.
+	StatusCode int
+	// The raw body if we couldn't understand it.
+	RawBody string
```

Comment:
I'd not expose `RawBody` until we actually have a use case for it.

Reply:
Done.